cmd/expiration-mailer/main.go

```diff
@@ -88,6 +90,8 @@ func (m *mailer) sendNags(contacts []*core.AcmeURL, certs []*x509.Certificate) e
 	domains = core.UniqueLowerNames(domains)
 	sort.Strings(domains)
 
+	m.log.Debug(fmt.Sprintf("Sending mail for %s (%s)", strings.Join(domains, ", "), strings.Join(serials, ", ")))
```

Comment:
nit: The way these are printed may be confusing since the first array is many-to-one and the second is one-to-one.